athenahealth/resources/GetPatient.json

```diff
@@ -107,7 +107,7 @@
         "contactpreference_billing_email": true,
         "mobilephone": "8608105503",
         "contactpreference_announcement_email": true,
-        "hasmobile": true,
+        "hasmobileyn": true,
```

Comment:
Given the conversation above do these GET responses need to be `hasmobile`?

Reply:
ah yea that should be updated, although it's not really being used